exchanges/subscription/subscription.go

```diff
@@ -31,6 +31,7 @@ const (
 	AllTradesChannel = "allTrades"
 	MyTradesChannel  = "myTrades"
 	MyOrdersChannel  = "myOrders"
+	WalletChannel    = "wallet"
```

Comment:
Wondering if this should be MyWalletChannel 🤷

Reply:
That's more consistent with what you have so sure.

Reply:
Fixed 415c88fd